ipc/cli/src/commands/checkpoint/bottomup_height.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: MIT
+
+use std::fmt::Debug;
+use std::str::FromStr;
+
+use async_trait::async_trait;
+use clap::Args;
+use ipc_sdk::subnet_id::SubnetID;
+
+use crate::commands::get_ipc_provider;
+use crate::{CommandLineHandler, GlobalArguments};
+
+/// The command to get the last bottom up checkpoint height in a subnet.
+pub(crate) struct LastBottomUpCheckpointHeight;
+
+#[async_trait]
+impl CommandLineHandler for LastBottomUpCheckpointHeight {
+    type Arguments = LastBottomUpCheckpointHeightArgs;
+
+    async fn handle(global: &GlobalArguments, arguments: &Self::Arguments) -> anyhow::Result<()> {
+        log::debug!(
+            "list bottom up checkpoint height with args: {:?}",
+            arguments
+        );
+
+        let provider = get_ipc_provider(global)?;
+        let subnet = SubnetID::from_str(&arguments.subnet)?;
+
+        let height = provider.last_bottom_up_checkpoint_height(&subnet).await?;
+        println!("height: {height}");
+
+        Ok(())
+    }
+}
+
+#[derive(Debug, Args)]
+#[command(about = "Last bottom up checkpoint height for a child subnet")]
```

Comment:
This is the last checkpoint committed in the parent, or the last height signed in the child? Is not clear to me. It would be good to clarify it.
```suggestion
#[command(about = "Last bottom up checkpoint height committed in a child subnet")]
```